src/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -279,6 +279,7 @@ public static string CustomWebUiRedirectUriMismatch(string expectedUri, string a
         public const string MatsAndTelemetryCallbackCannotBeConfiguredSimultaneously = "MATS cannot be configured at the same time as a TelemetryCallback is provided. These are mututally exclusive.";
         public const string AkaMsmsalnet3BreakingChanges = "See https://aka.ms/msal-net-3-breaking-changes";
 
-        public const string B2CHostNotTrusted = "The B2C authority that was used is not in the trusted host list. ";
+        public const string B2CAuthorityHostMisMatch = "The B2C authority host that was used in the config object is not the same authority host used in the AcquireToken call. " +
```

Comment:
same here.  i don't think the "config object" means as much to customers as it does to us.